src/rdpc/fetchAnalyses.ts

```diff
@@ -75,8 +77,11 @@ const fetchAnalyses = async ({
   analysisType: string;
   from: number;
   size: number;
+  egoJwtManager: EgoJwtManager;
   donorId?: string;
 }): Promise<Analysis[]> => {
+  const jwt = (await egoJwtManager.getLatestJwt()) as EgoAccessToken;
```

Comment:
since `egoJwtManager` is required now, this can just be `const jwt = await egoJwtManager.getLatestJwt()`